pages/common/gpg-zip.md

```diff
@@ -0,0 +1,15 @@
+# gpg-zip
+
+> Encrypt files and directories in an archive using GPG.
+
+- Encrypt a directory into archive.gpg using a passphrase:
+
+`gpg-zip --symmetric --output {{archive.gpg}} {{dirname}}`
```

Comment:
Perhaps `path/to/directory` instead of `dirname` would make it more consistent with existing pages?